src/compiler/compile/Component.ts

```diff
@@ -797,7 +797,13 @@ export default class Component {
 					const deep = assignee.type === 'MemberExpression';
 
 					names.forEach(name => {
-						if (scope.find_owner(name) === instance_scope) {
+						const scope_owner = scope.find_owner(name);
```

Comment:
Could this be DRYed up a bit? We seem to be doing the same thing whether we find the variable in the module scope or the instance scope.

Reply:
updated!